praw/const.py

```diff
@@ -26,6 +26,7 @@
     'contest_mode':           'api/set_contest_mode/',
     'del':                    'api/del/',
     'deleteflair':            'r/{subreddit}/api/deleteflair',
```

Comment:
I know this isn't your fault, but it looks like `deleteflair` is in the wrong location. Would you mind moving it to where it should go?